resources/default-services/jukebox-daemon.service

```diff
@@ -1,15 +1,17 @@
 [Unit]
 Description=Jukebox Daemon
 # pulseaudio.service is required during shut down of this service, as we play shutdown sound (and adjust the volume)
-After=network.target sound.target mpd.service pulseaudio.service
-Requires=mpd.service pulseaudio.service
+After=network.target sound.target
 
 [Service]
 ExecStart=/usr/bin/python3 run_jukebox.py
 WorkingDirectory=/home/pi/RPi-Jukebox-RFID/src/jukebox
 StandardOutput=inherit
 StandardError=inherit
 Restart=always
+User=pi
+Group=pi
```

Comment:
Since version 3.1.0 the Jukebox Service is started as user-local service from the user pi (not as system service). As such it already runs under the user pi. What is the additional benefit of specifying User/Group here? 
Is privilege escalation still possible? From within the service, we use `sudo reboot / shutdown` at the moment. Would these still work?

Reply:
you are right this should be done automatically without specifying the user here